source/shared-mixins/globals.jade

```diff
@@ -19,12 +19,12 @@ mixin breadcrumbs
 mixin username-input
     label(for='username')
         | Username
-    input(type='text', id='username', name='username', placeholder='Username',value='#{viewModel?viewModel.username:""}')
+    input(type='text', id='username', name='username', placeholder='Username',value='#{viewModel?viewModel.username:""}',required)
```

Comment:
Yes, ti will be enforced on compliant browsers 
http://www.w3schools.com/tags/att_input_required.asp